release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2143,6 +2143,15 @@ $ oc get etcd '-o=jsonpath={range .items[0].status.conditions[?(@.type=="NodeIns
 * The web console downloads deployment will fail on nodes configured with `ipv6.disable=1`.
  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1795325[*BZ1795325*])
 
+* An issue in Topology Manager could result in NUMA resources not being aligned to the same NUMA node if Guaranteed QoS pods are created simultaneously on the same node.
+As a result, the resources requested in the pod spec might not be NUMA aligned. This issue is fixed in {product-title} 4.5.
```

Comment:
I would avoid stating the fix for an unreleased version, unless you feel strongly this is absolutely necessary. It will be documented as a bug fix in the 4.5 RNs.